### DISCUSSION TITLE
Mantine on React Native?

### DISCUSSION THREAD

**Discussion by markotomcik** [mantinedev/mantine#1671] [Feedback]
I'm going to do one project in React Native and I thought it would be great if sometime the Mantine was ever available there.

**Comment by rtivital** [comment]
We are not planning react native support, it is the entire different project that will take a lot of time – I do not use react native and thus it does not make much sense to spend months of full time work on this.

**Reply by Louai-Zokerburg** [reply]
@rtivital I guess after few years from saying this statement and after that Mantine ui got some interested and love from developers it would be a wise move to start considering supporting more platforms, its a damage for such a great and beautiful project  to only preserved for react.

**Reply by MajorWookie** [reply]
@Louai-Zokerburg Check out https://tamagui.dev/. It seems to be the closest to Mantine in philosophy.

**Comment by oxdrtech** [comment]
Hoje sempre que eu tenho um projeto novo eu construo o projeto usando a Mantine, seria muito interessante poder usar toda a praticidade, e aparecia bonita nos meus layouts tambem nos aplicativos

**Comment by MajorWookie** [comment]
@markotomcik check out https://tamagui.dev/

**Comment by shaadcode** [comment]
This idea is very interesting. Considering that it has been a long time since this idea was presented, I think it should be reconsidered.
mantine has great potential and power.

**Comment by IsaacInsoll** [comment]
Yep, I've just started work on a React Native version of my Mantine web-app, would love to have consistent UI between web and mobile platforms
